VSTS/Tasks/SonarQubeScannerMsBuildBegin/SonarQubeHelper.ps1

```diff
@@ -0,0 +1,334 @@
+# When passing arguments to a process, the quotes need to be doubled and   
```

Comment:
I am quite new to powershell and the marketplace extensions so I am trying to build something in an incremental way. 
1/ Stupid copy/paste. 
2/ Adds unit-tests.
3/ Optimize code
I will not hesitate to contact you for help if I get stuck. Thanks!